packages/components/psammead-caption/package.json

```diff
@@ -18,10 +18,10 @@
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-caption/README.md",
   "dependencies": {
     "@bbc/gel-foundations": "^3.0.0",
-    "@bbc/psammead-styles": "^0.3.2"
+    "@bbc/psammead-styles": "^1.0.1"
```

Comment:
`"version": "1.1.7",` also needs updating to `"version": "1.1.8",` and `npm i` needs running to update `package-lock.json` version number